test_data/indent_string_literal_interpolation.good.nix

```diff
@@ -44,19 +44,6 @@
       bla = hoi
     '';
 
-  singleAsciiDoc = value: ''
-    ${
-      if lib.hasAttr "example" value
-      then ''
-        Example::
-        ${
-          builtins.toJSON value.example
-        }
-      ''
-      else "No Example:: {blank}"
-    }
-  '';
-
```

Comment:
removing this to avoid indempotent failed in the test, because it seems the failure is not because of the new string rule. So, I will add this test again in the other branch.